### ISSUE TITLE
Feature request: Hide tooltip

### ISSUE THREAD

**Issue by teiws** [lygstate/NotepadStarter#1]
Hi.
Could you make it possible to hide the tooltip permanently from the tray icon?

**Comment by lygstate** [comment]
do you means only appear a single tooltip or something else?

**Comment by teiws** [comment]
Yes, have an option to only display the tooltip one time, on the first use of NotepadStarter, and then never again (even after closing, reboot, shutdown etc.).
Thanks in advance!

**Comment by lygstate** [comment]
Then I would setting the default and only option that there is always a single tooltip.

**Comment by Blazein** [comment]
Hi teiws, I have the same issue using NotepadStarter under windows 10. The plugin launches notepad++ fine, but causes a windows tooltip popup and system notification sound every time you edit a file. It is quite annoying. I've tried installing NotepadStarter with the ":install-replace" method, but it doesn't seem to work at all.

**Comment by Blazein** [comment]
Hi @teiws, I have the same issue using NotepadStarter under windows 10. The plugin launches notepad++ fine, but causes a windows tooltip popup and system notification sound every time you edit a file. It is quite annoying. I've tried installing NotepadStarter with the ":install-replace" method, but it doesn't seem to work at all.

**Comment by lygstate** [comment]
How about to disable the tooltip permanently?

**Comment by Blazein** [comment]
I like my tooltips, but I don't need them popping up every time I edit a file. Get it?

Either way, I'll find another solution for replacing notepad.exe or none at all. Thanks!

**Comment by zapy85** [comment]
i have no issues with popups with this plugin on windows 10!
i however had issues with another launcher called notepad++ launcher causing popups but this plugin does not do that, are you sure you are in the right forum @Blazein ? did you install this plugin from the notepad++ plugin manager?

**Comment by ALIENQuake** [comment]
@lygstate I have the same issue, those popups are very annoying. It should have show one popup after first installation and the never again. Or just disable this popup completely.

**Comment by stevencohn** [comment]
I had the same issue so I forked this repo and modified the code to suppress the system tray notification balloon. You can find it here: https://github.com/stevencohn/NotepadStarter along with a Win32 release zip.
